place_tracker/web/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta content="IE=Edge" http-equiv="X-UA-Compatible">
+  <meta name="description" content="A new Flutter project.">
+
+  <!-- iOS meta tags & icons -->
+  <meta name="apple-mobile-web-app-capable" content="yes">
+  <meta name="apple-mobile-web-app-status-bar-style" content="black">
+  <meta name="apple-mobile-web-app-title" content="place_tracker">
+  <link rel="apple-touch-icon" href="icons/Icon-192.png">
+
+  <!-- Favicon -->
+  <link rel="shortcut icon" type="image/png" href="favicon.png"/>
+
+  <!-- // Other stuff -->
+
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBEKgpAdjMVMv9IKX-BBsjzwgYMxfZonbw"></script>
```

Comment:
Is the key intended to be in here? We don't ship the app with the mobile API keys, which is why I'm asking. Even if this one isn't as "secret" as the mobile versions, it might be better to disassociate the app from any one account and update the README to tell people where to stick their own key.

Reply:
Normally I would do that, but If we take it out, we can't deploy it to http://flutter.github.io/samples anymore. 

We could consider writing this string as part of the samples index deployment (perhaps via an environment variable given to a GitHub action.) but the API key is still visible if you view-source on the page.

Reply:
Added a section to the README for now

Reply:
The key can (and should) be "restricted" to the domains where the app is deployed, so it doesn't matter if anybody sees the key and attempts to use it, it won't work for them. I've done this before, I can help with the setup.

(Note that it doesn't matter if you move it to github secrets or not, once the website is published, the key will be in the DOM and then it can be copied from there)

Mobile keys are another story, probably, not sure how much they can be restricted to a single app :P

Reply:
Sounds good, I added some domain restrictions:

<img width="571" alt="Screen Shot 2020-09-23 at 11 05 43 AM" src="https://user-images.githubusercontent.com/1145719/94051718-c9bbd380-fd8c-11ea-8789-c7733d2ac418.png">

Reply:
Deleted my previous comment because I just stated the obvious (again) xD